src/pages/tutorials/frontend/create-custom-widget.md

```diff
@@ -153,10 +153,15 @@ And finally, create the template that will be used for showing the widget's data
 
 ```php
 <?php
-/** \ExampleCorp\Learning\Block\Widget\Test $block */
+/** @var \Magento\Framework\View\TemplateEngineInterface $this */
+/** @var \ExampleCorp\Learning\Block\Widget\Test $block */
+/** @var \Magento\Csp\Api\InlineUtilInterface $csp */
+/** @var \Magento\Framework\Escaper $escaper */
+/** @var \Magento\Framework\Locale\LocaleFormatter $localeFormatter */
```

Comment:
I suggest removing this one, since it is available starting 2.4.5 only, and might confuse some developers.
```suggestion
```

Reply:
@eduard13 yea, you are right. Done